plugins/nf-ga4gh/src/main/nextflow/ga4gh/tes/executor/TesTaskHandler.groovy

```diff
@@ -171,26 +172,30 @@ class TesTaskHandler extends TaskHandler {
     }
 
     protected TesTask newTesTask() {
-        // the cmd list to launch it
-        def job = new ArrayList(BashWrapperBuilder.BASH) << wrapperFile.getName()
-        List cmd = ['/bin/bash','-c', job.join(' ') + " &> $TaskRun.CMD_LOG" ]
+        if( !task.container )
+            throw new ProcessUnrecoverableException("Process `${task.lazyName()}` failed because the container image was not specified")
 
-        def exec = new TesExecutorModel()
-        exec.command = cmd
+        final exec = new TesExecutorModel()
+        exec.command = List.of('/bin/bash', '-c', "${BashWrapperBuilder.BASH} ${wrapperFile.getName()} &> ${TaskRun.CMD_LOG}".toString())
```

Comment:
@vsmalladi here is the change made that may require a small change in how Task Commands are handled by the executor:

Old Task Command:
```sh
/bin/bash -c /bin/bash -ue .command.run &> .command.log
```

New Task Command (with a nested List):
```sh
/bin/bash -c [/bin/bash -ue] .command.run &> .command.log
```

Reply:
Tahnks I think i got that in the Azure PR as well.

Reply:
https://github.com/nextflow-io/nextflow/blob/5e9ea644ff34c49ae584b3bbfeca51ef55a76e97/plugins/nf-ga4gh/src/main/nextflow/ga4gh/tes/executor/TesTaskHandler.groovy#L194

I have it now for Azure. Should we just make it for all TES ?